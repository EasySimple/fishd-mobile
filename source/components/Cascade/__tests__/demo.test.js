import webDemoTest from '../../../tests/demoTest';

webDemoTest('Cascade');
