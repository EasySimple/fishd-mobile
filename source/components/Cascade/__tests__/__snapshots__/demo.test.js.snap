// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Cascade demo correctly 1`] = `
<div>
  <div>
    <div
      class="fm-cascade-wrapper"
    >
      <div
        class="fm-cascade-select"
        tabindex="-1"
      >
        请选择
      </div>
      <div
        style="position:absolute;top:0;left:0;width:100%"
      >
        <div
          class="cascade-column"
        >
          <div
            class="col selectCol fullWidth"
          >
            <div
              class="item"
            >
              label-1
              <i
                class="icon iconfont fm-icon fm-icon-allow-right"
                style="font-size:12px"
              />
            </div>
            <div
              class="item"
            >
              label-2
              <i
                class="icon iconfont fm-icon fm-icon-allow-right"
                style="font-size:12px"
              />
            </div>
          </div>
          <div
            class="col fullWidth"
          />
        </div>
      </div>
    </div>
    <div
      class="fm-cascade-wrapper"
    >
      <div
        class="fm-cascade-select"
        tabindex="-1"
      >
        请选择
      </div>
      <div
        style="position:absolute;top:0;left:0;width:100%"
      >
        <div
          class="cascade-column"
        >
          <div
            class="col selectCol fullWidth"
          >
            <div
              class="item"
            >
              label-1
              <i
                class="icon iconfont fm-icon fm-icon-allow-right"
                style="font-size:12px"
              />
            </div>
            <div
              class="item"
            >
              label-2
              <i
                class="icon iconfont fm-icon fm-icon-allow-right"
                style="font-size:12px"
              />
            </div>
          </div>
          <div
            class="col fullWidth"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
